src/fusor/translator.py

```diff
@@ -141,6 +141,21 @@ def _get_gene_element(self, genes: str, caller: Caller) -> GeneElement:
         ge = self.fusor.gene_element(gene=gene)
         return ge[0] if ge[0] else self._get_gene_element_unnormalized(gene)
 
+    def _fusion_symbol_check(self, gene_5prime: str, gene_3prime: str) -> bool:
```

Comment:
This could probably have a better name. Maybe like `_is_fusion_genes_diff` or something